docs/validators-api.md

```diff
@@ -260,3 +260,110 @@ Arguments:
 | ""                    | "20"              | True      |
 | ""                    | "21"              | False     |
 | "Manhattan"           | "21"              | True      |
+
+## hasNumber
+
+hasNumber validates if this field contains a minimum of one integer
+
+```jsx
+import { hasNumber } from "redux-freeform";
+
+const formConfig = {
+  password: {
+    validators: [hasNumber()]
+  }
+};
+```
+
+| Value         | Validates |
+| ------------- | --------- |
+| "password"    | False     |
```

Comment:
can you add an empty string to each of these truth tables? I think we have that in all of the others. Since it's an edge case, it's something I want to make clear in the docs for all validators